### ISSUE TITLE
Make `verdi export inspect` fast by default

### ISSUE THREAD

**Issue by ltalirz** [aiidateam/aiida-core#4531]
### Is your feature request related to a problem? Please describe

At the moment `verdi export inspect` defaults to a statistics overview that requires loading the `data.json` as well.

```
$ verdi export inspect  export_pyrene_mofs_22Sep20.aiida
--------------  ------
Version aiida   1.3.1
Version format  0.9
Computers       3
Groups          62
Links           159872
Nodes           56832
Users           1
--------------  ------
```

While it is certainly useful to have the possibility of getting such statistics, it also makes the command unnecessarily slow for large export files.
Many use cases may already be served by just printing information available from the export metadata, such as export version, aiida version used to generate the archive, timestamp when it was created (note: the latter information is currently missing from the archive format).

### Describe the solution you'd like

Move the statistics view to a `--statistics` flag.

Default to information that can be parsed from the metadata.json  and print at the bottom of the screen the command to get the extra statistics, e.g.
```
$ verdi export inspect  export_pyrene_mofs_22Sep20.aiida
--------------  ------
Version aiida   1.3.1
Version format  0.9
--------------  ------

Run `verdi export inspect  export_pyrene_mofs_22Sep20.aiida` for statistics on the archive contents
```

P.S. This concerns the current archive format. I suggest we put this on hold for now and reevaluate this once the new archive format is in place - if the statistics are super fast, we may not need to implement this change.

**Comment by chrisjsewell** [comment]
> if the statistics are super fast, we may not need to implement this change.

That's the hope 😄
